src/platform/forms-system/src/js/review/submit-states/ClientError.jsx

```diff
@@ -11,6 +11,7 @@ import scrollTo from 'platform/utilities/ui/scrollTo';
 
 export default function ClientError(props) {
   const { buttonText, formConfig, onBack, onSubmit, testId } = props;
+  const ariaDescribedBy = formConfig.ariaDescribedBy.submitSection;
```

Comment:
I think [optional-chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) should be added here, and the other submit-state files, to prevent an undefined JS error here, i.e. `formConfig.ariaDescribedBy?.submitSection`.

Reply:
I cleaned up naming to be more simplistic eliminating deeper chaining removing complexity I don't think we need, i.e.:

```
ariaDescribedBy = formConfig.ariaDescribedBySubmit;
```